src/toggle.jsx

```diff
@@ -79,6 +79,7 @@ let Toggle = React.createClass({
       },
       label: {
         color: this.props.disabled ? this.getTheme().labelDisabledColor : this.getTheme().labelColor,
+        width: 'calc(100% - 46px)',
```

Comment:
Yes, it is correct and it will work. Before it was "calc(100% - 60px)" inherited from EnhancedSwitch. 46px means 36px of toggle width + 8px of margin from left (in EnhancedSwitch) + 2px of padding-left from line 44.